apps/docs/src/app/core/component-docs/combobox/combobox-header/combobox-header.component.html

```diff
@@ -1,7 +1,15 @@
 <header>Combobox</header>
 <description>
-    The combobox component is an opinionated composition of the input-group and popover components to accomplish the UI
-    pattern of an autocomplete-like experience, also known as combobox input.
+    <p>
+        The combobox component is an opinionated composition of the input-group and popover components to accomplish the UI
+        pattern of an autocomplete-like experience, also known as combobox input.
+    </p>
+
+    <fd-message-strip [dismissible]="false" type="information">
+        <b>IMPORTANT</b> Combobox uses <a href="/fundamental-ngx#/core/popover" target="_blank">Popover</a>.
+        Therefore, if you place Combobox inside scrollable container - you need to add <code>cdk-scrollable</code> (or <code>cdkScrollable</code>) directive into container
```

Comment:
I would suggest to mention just cdkScrollable, since it's referred in official angular documentation

Reply:
agree

Reply:
Ok. In this case I need to change one more place.

Reply:
Actually I am disagree with us) For several reasons:
1) We already have one place with this method, I just did the same.
2) CDK uses cdkScrollable on overview tab, but on api tab there is full info.
3) As for me, it's better to add more text and make it less beautiful, but have complete information so as not to confuse users.

But If you both considered the same, I will do the same.